### ISSUE TITLE
Stuck after spawn

### ISSUE THREAD

**Issue by fabian-nunes** [frida/frida#2373]
Hello, i am trying to execute some tutorial SSL pining demos however after executing the frida command, its stuck forever in spawning.

I am trying this on a rooted Samsung A40 connected through USB. I already updated my frida however the problem persists.

![Screenshot_43](https://user-images.githubusercontent.com/54534495/210264698-dab63c9b-bf2b-4889-9fd3-cdbaebc34169.png)

First i had a problem related to the frida gadget, i had to download the file from the repo and put it in the folder shown in the terminal.

I already tried Frida on macOS and Windows and the result is the same, can somebody help me?

<img width="1146" alt="Captura de ecrã 2022-12-28, às 13 23 37" src="https://user-images.githubusercontent.com/54534495/210264850-17ce715d-6ae0-4360-9b77-caba34e6415d.png">

**Comment by ozansefa** [comment]
Have you found the solution to this problem?

**Comment by fabian-nunes** [comment]
Yes i ahad to download the frida gadgets and put it on the path mentioned, however, I had other problems unrelated to this topic and needed to use a specific version (15.2) since the 16 did not work with my rooted Samsung. And use the command ` setprop persist.device_config.runtime_native.usap_pool_enabled false`.

**Comment by PRO100CHOK** [comment]
Hi.Frida does not hang on "Spawning" what to do? 
Tried: update frida to the latest version, use "setprop persist.device_config.runtime_native.usap_pool_enabled false" command

Task: use the script to remove the certificate https://codeshare.frida.re/@akabe1/frida-multiple-unpinning/

Samsung Galaxy a50
![image](https://github.com/frida/frida/assets/51245970/6cdfcf1a-0bf9-45f1-9b14-6df843382e58)

**Comment by jambuz** [comment]
I managed to fix the issue by ensuring that Frida is running on the target device (rooted OnePlus 6) via the [magisk-frida](github.com/ViRb3/magisk-frida) Magisk module. After that, Frida no longer got stuck on the ``Spawning `com.package.name` `` phase.

**Comment by jambuz** [comment]
I managed to fix the issue by ensuring that Frida is running on the target device (rooted OnePlus 6) via the [magisk-frida](https://github.com/ViRb3/magisk-frida) Magisk module. After that, Frida no longer got stuck on the ``Spawning `com.package.name` `` phase.

**Comment by IrvanWijayaSardam** [comment]
having the same issue, but i'm using KernelSU, is anyone having the same issue ?

**Comment by batutahibnu17** [comment]
now I facing the issue on Pixel 6a, I don't know why this time happening
frida version 16.1.4 with Magisk-Frida same version too
![Image](https://github.com/user-attachments/assets/21baaa67-d032-402d-81aa-e105d3fa0c16)
